libraries/aws_ec2_instance.rb

```diff
@@ -1,16 +1,17 @@
 # author: Christoph Hartmann
-
-class Ec2 < Inspec.resource(1)
-  name 'aws_ec2'
+class AwsEc2Instance < Inspec.resource(1)
+  name 'aws_ec2_instance'
   desc 'Verifies settings for an EC2 instance'
 
   example "
-    describe aws_ec2('i-123456') do
+    describe aws_ec2_instance('i-123456') do
       it { should be_running }
+      it { should have_roles }
     end
 
-    describe aws_ec2(name: 'my-instance') do
+    describe aws_ec2_instance(name: 'my-instance') do
       it { should be_running }
+      it { should have_roles }
```

Comment:
So we are just checking if we have 'some' role defined. Are we also going to have the 'roles' in the @params so we can say:

```its('roles') { should include 'role1, role2' } ```

Reply:
We are not planning to implement the behaviour you describe as part of this PR, @aaronlippold .  It sounds like a fine idea, though, and I don't see any inconsistencies between this PR's `has_roles?` method and your proposed `roles` method.